### ISSUE TITLE
FOR AN EXAMPLE : CHECK FOR THE CRITICAL ERRORS IN SYSTHEM LOG

### ISSUE THREAD

**Issue by ffcweb** [ffcweb/Example_for_w3#2]
This is the list of issues for our health-checks projects. For now, it only has one issue asking us to update our documentation. One of our colleagues suggested that we create a new health check that verifies if there are any critical error messages inside the system logs like current log or syslog. Errors that appear there might help troubleshoot some interesting problems. So, this sounds like a good idea for a new check. Let's create an issue for this feature by clicking on the New issue button. For the title of the issue, we'll say that we want to check for critical errors in system logs. And for the issues description we'll say that the new check should go through var/log/currentlog. And var/log/syslog and check if there are any critical errors that need attention.

When writing in issues description, it's a good idea to include all the information that we have about the problem or missing feature and any ideas on how to solve it. And if new information comes up later on, it can be added as additional comments on the same issue.

**Comment by ffcweb** [comment]
For example, if we use #2 in a comment, it will automatically reference the issue we just created.
If you're fixing an issue through a pull request, it's possible to automatically close the issue directly once the code is merged.

**Comment by ffcweb** [comment]
If you're fixing an issue through a pull request, it's possible to automatically close the issue directly once the code is merged.
To do this, you need to include a string like closes:#4 in your commit message or as a part of the description of your pull request.

Once the code gets merged into the main tree, GitHub will automatically close the issue with a message linking it to the new commit.
